src/library.rs

```diff
@@ -844,6 +844,20 @@ impl Library {
                         }
                     }
                 }
+                if let Some(tid) = env.library.find_type(0, &full_name) {
+                    let gobject_id = env.library.find_type(0, "GObject.Object").unwrap();
+
+                    for &super_tid in env.class_hierarchy.supertypes(tid) {
+                        if super_tid != gobject_id &&
+                           env.type_status(&super_tid.full_name(&env.library)).ignored() {
+                            println!("[NOT GENERATED PARENT] {}.{} for {}.{}",
```

Comment:
Didn't we have macros for errors, warnings, debug, etc?

Reply:
We do but I just did like I did previously for this mode.